packages/preview/delegis/0.1.0/template/main.typ

```diff
@@ -0,0 +1,60 @@
+#import "@preview/delegis:0.1.0": *
+
+#show: it => delegis(
+  // Metadata
+  title: "Vereinsordnung zu ABCDEF",
+  abbrev: "ABCDEFVO",
```

Comment:
```suggestion
  abbreviation: "ABCDEFVO",
```

Reply:
Good catch – thanks! Sorry, that one slipped through the cracks.